build_tools/docker/dockerfiles/base.Dockerfile

```diff
@@ -69,7 +69,7 @@ RUN ./install_bazel.sh && rm -rf /install-bazel
 WORKDIR /install-python
 
 # Minimum supported Python version
-ARG PYTHON_VERSION=3.7
+ARG PYTHON_VERSION=3.8
```

Comment:
You should rebuild all images that depend on base, not just the "frontends" images after this change. See https://github.com/openxla/iree/tree/main/build_tools/docker#dependencies-between-images

Reply:
Scott is correct. I don't believe this has been fixed afterwards either and our Docker images are in a weird unsynchronized state.

Reply:
(Fixed in https://github.com/openxla/iree/pull/12857)

Reply:
Ah thanks Scott. I saw that in history and thought it was this PR